polygon_cli/polygon_cli.py

```diff
@@ -15,6 +15,15 @@
 from .exceptions import PolygonNotLoginnedError
 
 parser = argparse.ArgumentParser(prog="polygon-cli")
+parser.add_argument('-v', '--verbose',
+                    action='store_true',
+                    dest='verbose',
+                    default=True,
+                    help='Verbose output')
+parser.add_argument('-nv', '--no-verbose',
```

Comment:
I think it's not very good naming for this option. In unix systems -nv is threated by many tools as too options -n and -v. I'm a bit surprised, python don't parse it in that way. Maybe -V is better?